README.md

```diff
@@ -8,6 +8,26 @@
 
 Here you can write and save your scores.
 
+## Setup
+
+### Clone this repository
+
+```bash
+$ git clone https://github.com/vladanvi99/leaderboard.git
+$ cd leaderboard
+```
+
+### Run project
```

Comment:
- [ ] On `Run project` section, you could add the way to use the project with `npm start`. 🐶 
- [ ] Please, add instructions about how to use the API (how to create your own game, how to use the API, credentials if is required, etc.). 🐱